app/models/vacancy.rb

```diff
@@ -265,6 +265,10 @@ def organisation
     organisation_vacancies.first&.organisation
   end
 
+  def schools
```

Comment:
You don't need to do this, we should just be interested in `vacancy.organisations`